### PR TITLE
✨ Make the host header configurable

### PR DESCRIPTION
**Pull request by peterfication**
... but fallback to `PROXY_URL`

### CODE REVIEW

File: nginx.proxy-cached.conf [hunk 1]
```diff
@@ -1,42 +0,0 @@
-proxy_cache_path /tmp/proxy-cache levels=1:2 keys_zone=default_cache:10m max_size=1g
```
Comment (peterfication) [inline nginx.proxy-cached.conf:1]: This file has not been used anymore as far as I can see.

**Review by smowafy — APPROVED** [review]

**Comment by smowafy** [comment]
One comment though for later, what about we prefix all our specific env vars? Because I feel for example that `HOST` is too general of a name

**Comment by peterfication** [comment]
Good idea! In this case `PROXY_HOST`?

**Comment by smowafy** [comment]
For example yea :+1:

**Review by smowafy — APPROVED** [review]

**Pull request merged**